app/models/report.rb

```diff
@@ -14,4 +20,20 @@ def editable?(target_user)
   def created_on
     created_at.to_date
   end
+
+  private
+
+  def build_mentions
+    reset_active_mentions
+    return unless (match_data = %r{localhost:3000/reports/(\d+)}.match(content))
+
+    mention = ReportMention.new
+    mention.source_report_id = id
+    mention.target_report_id = match_data[1].to_i
+    mention.save
+  end
+
+  def reset_active_mentions
+    active_mentions.each(&:destroy)
```

Comment:
```suggestion
    active_mentions.destroy_all
```

でいいですね。
そして、これ→ https://bootcamp.fjord.jp/pages/when-you-should-define-method

Reply:
ここは実装時にメソッド化するか大分悩みました、`build_mentions`という名前にも関わらず最初に実行されるのが'active_mentions.each(&:destroy)'なので後からコードを読み返す際に（自身でも自身以外でも）あれ？ってなるかなと思いメソッド名でリセットしてることを明示すれば多少理解がしやすくなるかなーとの思いでメソッド化したのですが、あまり良い手段ではなかったみたいですね。

ご指摘頂いた方法に変更しました！